src/utilities/superAdminCreation.ts

```diff
@@ -0,0 +1,8 @@
+import { User } from "../models";
+
+export const updateUserToSuperAdmin = async (email: string) => {
```

Comment:
A seperate function just for this is wasteful. Remove this, and do this while creating the user itself.

Reply:
My idea for creating a separate function - 

Sign up function will mostly be used when user is created, by adding conditions to it we are making it bulky and adding extra responsibility. On the other hand if we put that logic in separate function it the responsibility is shared among them and in future if we add any other fields particularly for super admin we won't need to change the sign up function. 
Let me know your thoughts on the same.

Reply:
@BhuvaneshPatil But there is no logic being duplicated here. What's the point of having one utility function that's used at exactly one place? Besides with this approach you're using two database calls to do the same thing, why waste a database call?

If the same logic is duplicated 3 or more times then it makes sense to seperate out into a seperate function.